test/Renci.SshNet.Tests/Classes/PrivateKeyFileTest.cs

```diff
@@ -1,4 +1,8 @@
-﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+﻿// Excluding these tests on net462 platform, beacuse using MD5 hash throws: System.InvalidOperationException:
+// 'This implementation is not part of the Windows Platform FIPS validated cryptographic algorithms.'
+#if !NET462_TARGET_FRAMEWORK
```

Comment:
I'll have to check this on my machine because it seems to be working now.

Reply:
@realvizu Can you revert this? It is not a part of this PR.

Reply:
Sure, reverted it.